contracts/utils/BotRefillsL2.sol

```diff
@@ -6,15 +6,21 @@ import "../auth/AdminAuth.sol";
 import "../interfaces/uniswap/v3/ISwapRouter.sol";
 import "../interfaces/uniswap/v3/IQuoter.sol";
 import "../interfaces/IBotRegistry.sol";
+import "../interfaces/chainlink/IAggregatorV3.sol";
 import "./TokenUtils.sol";
 import "./helpers/UtilHelper.sol";
 import "./FeeRecipient.sol";
 
-contract BotRefillsL2 is AdminAuth, UtilHelper{
+
+contract BotRefillsL2 is AdminAuth, UtilHelper {
     using TokenUtils for address;
     error WrongRefillCallerError();
     error NotAuthBotError();
 
+    uint256 public constant ALLOWED_SLIPPAGE = 2e16;
+    IAggregatorV3 internal constant daiFeed = IAggregatorV3(CHAINLINK_DAI_FEED);
```

Comment:
Can we use FeedRegistry as we have one of ours on Optimism/Arbitrum (check defisaver-v3-contracts/addresses/pricefeeds)

Reply:
makes sense